src/rest/QueryAnswers.ts

```diff
@@ -0,0 +1,10 @@
+export interface IQueryQuestionAnswer {
```

Comment:
Why is the filename QueryAnswers and the interface QueryQuestionAnswer?

Reply:
Woops, I've no idea. Fixing this.